genie/input_to_database.py

```diff
@@ -259,13 +259,10 @@ def validatefile(syn, entities, validation_statusdf, error_trackerdf,
     # Need to figure out to how to remove this
     # This must pass in filenames, because filetype is determined by entity name
     # Not by actual path of file
-    filetype = validate.determine_filetype(syn, filenames, center)
+    validator = validate.Validator(syn=syn, center=center, filepathlist=filepaths)
```

Comment:
this should really be explicitly parameterized with the format registry dict instead of using the default.

Reply:
done